schema/interface.ipv4.yml

```diff
@@ -200,6 +200,22 @@ properties:
     description: "Enables DHCP Snooping on a VLAN"
     type: boolean
     default: false
+  ip-arp-inspect-vlan-config:
```

Comment:
Going forward perhaps we could consider putting objects into their own yml file, i.e.
interface.ipv4.yml would then have:
ip-arp-inspect-vlan-config:
  $ref: "https://ucentral.io/schema/v1/interface/ipv4/ip-arp-inspect-vlan-config/"
  
The portion below would then be contained in a new file, interface.ipv4.ip-arp-inspect-vlan-config.yml

The end composition would not be changed but it might be easier to keep track of things, some of these files are getting quite long